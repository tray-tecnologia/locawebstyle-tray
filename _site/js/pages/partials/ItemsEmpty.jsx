import React from 'react';
import { Link } from 'react-router';

class ItemsEmpty extends React.Component {

    componentDidMount() {
        locastyle.general.init();
    }

    render () {
        return (
            <div>
                <div className="ls-lg-margin-bottom">
                    <h2 className="ls-title-5">Você ainda não possui itens cadastrados.</h2>
                </div>

                <div className="ls-box ls-lg-space">
                    <h3 className="ls-title-3 ls-sm-margin-bottom">Inclua seu primeiro item</h3>

                    <p className="ls-md-margin-bottom">
                        Eiusmod elit magna irure laborum culpa culpa anim tempor velit. Tempor aliqua incididunt in est ex qui. Laboris commodo magna duis id dolor ea. Elit id officia ipsum qui anim veniam officia esse cupidatat veniam qui irure eiusmod aute. Proident ea culpa qui tempor consectetur. Tempor sint incididunt non officia voluptate qui enim duis excepteur est adipisicing voluptate.
                    </p>

                    <Link
                        className="ls-btn-primary ls-sm-margin-top"
                        id="btn-action"
                        to="/items/add">
                        Incluir item
                    </Link>

                    <button id="btn-default" className="ls-btn ls-sm-margin-top" onClick={this.props.addItems} data-toggle-class="ls-display-none" data-target="#btn-default, #btn-default-disabled">
                        Carregar itens
                    </button>
                    <button id="btn-default-disabled" className="ls-btn ls-sm-margin-top ls-disabled ls-display-none">
                        Aguarde, carregando itens...
                    </button>
                </div>
            </div>
        );
    }

}

export default ItemsEmpty;
