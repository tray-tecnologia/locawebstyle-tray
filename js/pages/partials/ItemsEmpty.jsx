import React from 'react';
import { Link } from 'react-router';

class ItemsEmpty extends React.Component {

    componentDidMount() {
        locastyle.general.init();
    }

    render () {
        return (
            <div>
                <div className="ls-lg-margin-bottom">
                    <h2 className="ls-title-5">Você ainda não possui itens cadastrados.</h2>
                </div>

                <div className="ls-box ls-lg-space">
                    <h3 className="ls-title-3 ls-sm-margin-bottom">Inclua seu primeiro item</h3>

                    <p className="ls-md-margin-bottom">
                        Mussum ipsum cacilds, vidis litro abertis. Consetis adipiscings elitis. Pra lá , depois divoltis porris, paradis.
                        Paisis, filhis, espiritis santis. Mé faiz elementum girarzis, nisi eros vermeio, in elementis mé pra quem é amistosis quis leo.
                        Manduma pindureta quium dia nois paga.
                    </p>

                    <Link
                        className="ls-btn-primary ls-sm-margin-top"
                        id="btn-action"
                        to="/items/add">
                        Incluir item
                    </Link>

                    <button id="btn-default" className="ls-btn ls-sm-margin-top" onClick={this.props.addItems} data-toggle-class="ls-display-none" data-target="#btn-default, #btn-default-disabled">
                        Carregar itens
                    </button>
                    <button id="btn-default-disabled" className="ls-btn ls-sm-margin-top ls-disabled ls-display-none">
                        Aguarde, carregando itens...
                    </button>
                </div>
            </div>
        );
    }

}

export default ItemsEmpty;
