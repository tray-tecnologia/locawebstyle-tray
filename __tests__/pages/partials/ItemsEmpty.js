import React from 'react';
import renderer from 'react-test-renderer';

import ItemsEmpty from '../../../js/pages/partials/ItemsEmpty.jsx';

locastyle.general.init = function() {};

describe('Items Empty', () => {
    it('show component', () => {
        let tree = renderer.create(
            <ItemsEmpty />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });

    it('show component', () => {
        let tree = renderer.create(
            <ItemsEmpty />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
