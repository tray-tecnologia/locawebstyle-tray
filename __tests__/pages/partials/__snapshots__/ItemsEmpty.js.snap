exports[`Items Empty show component 1`] = `
<div>
  <div
    className="ls-lg-margin-bottom">
    <h2
      className="ls-title-5">
      Você ainda não possui itens cadastrados.
    </h2>
  </div>
  <div
    className="ls-box ls-lg-space">
    <h3
      className="ls-title-3 ls-sm-margin-bottom">
      Inclua seu primeiro item
    </h3>
    <p
      className="ls-md-margin-bottom">
      Mussum ipsum cacilds, vidis litro abertis. Consetis adipiscings elitis. Pra lá , depois divoltis porris, paradis. Paisis, filhis, espiritis santis. Mé faiz elementum girarzis, nisi eros vermeio, in elementis mé pra quem é amistosis quis leo. Manduma pindureta quium dia nois paga.
    </p>
    <a
      className="ls-btn-primary ls-sm-margin-top"
      id="btn-action"
      onClick={[Function bound handleClick]}
      style={Object {}}>
      Incluir item
    </a>
    <button
      className="ls-btn ls-sm-margin-top"
      data-target="#btn-default, #btn-default-disabled"
      data-toggle-class="ls-display-none"
      id="btn-default"
      onClick={undefined}>
      Carregar itens
    </button>
    <button
      className="ls-btn ls-sm-margin-top ls-disabled ls-display-none"
      id="btn-default-disabled">
      Aguarde, carregando itens...
    </button>
  </div>
</div>
`;
